.github/workflows/ci.yml

```diff
@@ -26,56 +26,77 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - name: Apt install deps
+        run: |
+          sudo apt update
+          sudo apt-get install xvfb freeglut3-dev libglu1-mesa
+
       - uses: conda-incubator/setup-miniconda@v2
         with:
           auto-update-conda: true
-          mamba-version: "*"
-          channels: mantid,mantid/label/nightly,conda-forge
-          environment-file: environment.yml
 
-      - name: Apt install deps
+      - name: Setup libmamba solver
         run: |
-          sudo apt update
-          sudo apt-get install xvfb freeglut3-dev libglu1-mesa
+          conda install -n base conda-libmamba-solver
+          conda config --set solver libmamba
+
+      - name: Install requirements
+        run: |
+          # --quiet should turn off progress bars to make logs more readable
+          conda env create --file environment.yml --quiet
+          conda activate PyRS
+          conda env update --file environment.yml --quiet
```

Comment:
Did you mean to install the same environment twice?

Reply:
Mindless copy/paste :man_facepalming: